### DISCUSSION TITLE
Is there any way to sownload videos in the format they are available?

### DISCUSSION THREAD

**Discussion by mahdihasan2006** [NickvisionApps/Parabolic#757] [Q&A] [answered]
Currently the app asks to select an format from the available ones but no option to skip this selection.

**Comment by nlogozzo** [comment, marked as answer]
There is an option in the app's preferences called "Disallow (or Disable) Conversions"

This will disable that format selection and just ask you whether you would like to download the Video or just Audio and then it will automatically select the appropriate format for you.

**Reply by mahdihasan2006** [reply]
Thanks. It would be good to have this option when starting to download something.
